src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py

```diff
@@ -0,0 +1,980 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.alter_column('sentry_release', 'version', self.gf(
```

Comment:
Looks like you need to regenerate this migration anyways to pick up the `ref` column change.